substrate/client/runtime-utilities/src/lib.rs

```diff
@@ -1,21 +1,28 @@
 // This file is part of Substrate.
 
 // Copyright (C) Parity Technologies (UK) Ltd.
-// SPDX-License-Identifier: Apache-2.0
-
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Substrate client runtime utilities.
+//!
+//! Provides simpler APIs for calling into a `frame` runtime WASM blob and
+//! return metadata that needs to be checked.
```

Comment:
Some nits:
- simpler - is it misspell? If not simpler than what?
- "needs to be checked" - is it important from this module perspective? I mean metadata can be used in anyway.

Reply:
The description is vague. I rephrased this here: [0ba6ecb](https://github.com/paritytech/polkadot-sdk/pull/6450/commits/0ba6ecb244c1ae43bfc6262e2c1143bf3f735410)